ndex2/client.py

```diff
@@ -857,6 +1167,29 @@ def set_network_sample(self, network_id, sample_cx_network_str):
     def set_network_system_properties(self, network_id, network_properties):
         """
         Set network system properties
+	
+	.. code-block:: python
+
+		from ndex2.nice_cx_network import NiceCXNetwork
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		network_id = 'INSERT NETWORK UUID HERE'
+
+		#check the network properties before set value
+		print(my_ndex.get_network_summary(network_id))
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client